docs/source/releases/v1_12_0.rst

```diff
@@ -540,3 +540,17 @@ All geoips_db plugins now accessed through class-based geoips_db interface.
 ::
 
   modified: geoips/plugins/modules/procflows/single_source.py
+
+Make JSON Plugin Registries Readable
```

Comment:
As long as this should be in the `v1_12_0` release notes, this looks like it's good to go to me!

Reply:
I think this was added to 1.12.0 a while back, and has propagated into a number of PRs... This chunk of release note should be removed from 1.12.0 before merge.